gm4_desire_lines/data/desire_lines/functions/player.mcfunction

```diff
@@ -0,0 +1,7 @@
+scoreboard players operation @s gm4_desire_lines += @s gm4_dl_sprint
```

Comment:
Just out of curiosity, what is the intent of this section?

Reply:
Both scoreboards get added together so walking and sprinting both create a path